### PR TITLE
9stx6/dw 442/display matic balance

### PR DESCRIPTION
**Pull request by Stx69**
Main implementation for display matic balance:
- add Matic token.svg & gif , Matic Abi , getBalanceOf of Matic.api
- wip debug Matic token price against USDC in hints when mouse course hover a icon of Matic in balances contex panel.
-> I would like to ask for help with debuging not sure why pricePerToken: tokensPrices[TokenTypes.Matic].toFixed(2), doesnt work well... :(

### CODE REVIEW

File: src/components/Icons/Icons.tsx [hunk 1]
```diff
@@ -196,6 +198,10 @@ export const GhstIcon = ({ className, width, height }: IconProps) => {
   return <img className={className} width={width} height={height} src={ghst} alt='ghst' />;
 };
 
+export const MaticIcon = ({ className, width, height }: IconProps) => {
```
Comment (dmutrokot) [inline src/components/Icons/Icons.tsx:201]: you have defined two icons, please remove one of them

File: src/api/matic.api.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { EthersApi } from './ethers.api';
```
Comment (dmutrokot) [inline src/api/matic.api.ts:1]: please add typescript types instead of `any`

**Review by dmutrokot — CHANGES_REQUESTED** [review]

File: src/api/matic.api.ts (hunk 2 above)
Comment (Stx69) [reply src/api/matic.api.ts:1]: string

File: src/components/Icons/Icons.tsx (hunk 1 above)
Comment (Stx69) [reply src/components/Icons/Icons.tsx]: deleted non used

**Review by dmutrokot — APPROVED** [review]

**Comment by dudendy** [comment]
resolves: #206

**Review by dudendy — APPROVED** [review]

**Pull request merged**